<?xml version="1.0" encoding="UTF-8"?>
<testng-results ignored="0" total="5" passed="5" failed="0" skipped="0">
  <reporter-output>
  </reporter-output>
  <suite started-at="2021-10-16T17:49:57 IST" name="Default suite" finished-at="2021-10-16T17:49:57 IST" duration-ms="122">
    <groups>
    </groups>
    <test started-at="2021-10-16T17:49:57 IST" name="Default test" finished-at="2021-10-16T17:49:57 IST" duration-ms="122">
      <class name="TestStringCalculator">
        <test-method is-config="true" signature="init()[pri:0, instance:TestStringCalculator@68c72235]" started-at="2021-10-16T17:49:57 IST" name="init" finished-at="2021-10-16T17:49:57 IST" duration-ms="12" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- init -->
        <test-method signature="negativeInputReturnsException()[pri:0, instance:TestStringCalculator@68c72235]" started-at="2021-10-16T17:49:57 IST" name="negativeInputReturnsException" finished-at="2021-10-16T17:49:57 IST" duration-ms="14" status="PASS">
          <exception class="java.lang.Exception">
            <message>
              <![CDATA[Negative input!]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.Exception: Negative input!
at Calculator.findInvalidInput(Calculator.java:31)
at Calculator.getSum(Calculator.java:20)
at Calculator.calculator(Calculator.java:14)
at TestStringCalculator.negativeInputReturnsException(TestStringCalculator.java:32)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:64)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:564)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:133)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:598)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:173)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:824)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:146)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at org.testng.TestRunner.privateRun(TestRunner.java:794)
at org.testng.TestRunner.run(TestRunner.java:596)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:377)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:371)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:332)
at org.testng.SuiteRunner.run(SuiteRunner.java:276)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1212)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1134)
at org.testng.TestNG.runSuites(TestNG.java:1063)
at org.testng.TestNG.run(TestNG.java:1031)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.Exception -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- negativeInputReturnsException -->
        <test-method signature="returnsValue()[pri:0, instance:TestStringCalculator@68c72235]" started-at="2021-10-16T17:49:57 IST" name="returnsValue" finished-at="2021-10-16T17:49:57 IST" duration-ms="6" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- returnsValue -->
        <test-method signature="returnsZero()[pri:0, instance:TestStringCalculator@68c72235]" started-at="2021-10-16T17:49:57 IST" name="returnsZero" finished-at="2021-10-16T17:49:57 IST" duration-ms="3" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- returnsZero -->
        <test-method signature="sumOfNumbers()[pri:0, instance:TestStringCalculator@68c72235]" started-at="2021-10-16T17:49:57 IST" name="sumOfNumbers" finished-at="2021-10-16T17:49:57 IST" duration-ms="5" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- sumOfNumbers -->
        <test-method signature="twoNumbersNewLineDelimiterReturnSum()[pri:0, instance:TestStringCalculator@68c72235]" started-at="2021-10-16T17:49:57 IST" name="twoNumbersNewLineDelimiterReturnSum" finished-at="2021-10-16T17:49:57 IST" duration-ms="2" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- twoNumbersNewLineDelimiterReturnSum -->
      </class> <!-- TestStringCalculator -->
    </test> <!-- Default test -->
  </suite> <!-- Default suite -->
</testng-results>
